syn/yosys-uhdm/build.sh

```diff
@@ -18,7 +18,6 @@ cd Surelog && \
 #Create aliases for gcc/gxx as `abc` uses them directly in Makefile
 alias gcc=x86_64-conda_cos6-linux-gnu-gcc
 alias gxx=x86_64-conda_cos6-linux-gnu-gcc
-cd yosys && git apply ../yosys-patches/* && cd ..
```

Comment:
Should we remove the directory?

Reply:
The directory is already removed in upstream repository: https://github.com/antmicro/yosys-uhdm-plugin-integration
and it is not present in this repository: https://github.com/hdl/conda-eda/tree/master/syn/yosys-uhdm